<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
<title>TestNG Report</title>
<style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}.invisible {display:none}</style>
</head>
<body>
<table>
<tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr>
<tr><th colspan="7">青果</th></tr>
<tr><td><a href="#t0">查询运费</a></td><td class="num">3</td><td class="num">0</td><td class="num attn">6</td><td class="num">1,265</td><td></td><td></td></tr>
</table>
<table id='summary'><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">青果</th></tr></tbody><tbody id="t0"><tr><th colspan="4">查询运费 &#8212; failed</th></tr><tr class="failedeven"><td rowspan="6">com.edu.test.TransFeeTest</td><td><a href="#m0">feeTest3</a></td><td rowspan="1">1545820360641</td><td rowspan="1">80</td></tr><tr class="failedeven"><td><a href="#m1">feeTest5</a></td><td rowspan="1">1545820360816</td><td rowspan="1">90</td></tr><tr class="failedeven"><td><a href="#m2">feeTest6</a></td><td rowspan="1">1545820360916</td><td rowspan="1">90</td></tr><tr class="failedeven"><td><a href="#m3">feeTest7</a></td><td rowspan="1">1545820361006</td><td rowspan="1">90</td></tr><tr class="failedeven"><td><a href="#m4">feeTest8</a></td><td rowspan="1">1545820361096</td><td rowspan="1">100</td></tr><tr class="failedeven"><td><a href="#m5">feeTest9</a></td><td rowspan="1">1545820361196</td><td rowspan="1">80</td></tr><tr><th colspan="4">查询运费 &#8212; passed</th></tr><tr class="passedeven"><td rowspan="3">com.edu.test.TransFeeTest</td><td><a href="#m6">feeTest</a></td><td rowspan="1">1545820360021</td><td rowspan="1">539</td></tr><tr class="passedeven"><td><a href="#m7">feeTest2</a></td><td rowspan="1">1545820360561</td><td rowspan="1">80</td></tr><tr class="passedeven"><td><a href="#m8">feeTest4</a></td><td rowspan="1">1545820360721</td><td rowspan="1">95</td></tr></tbody>
</table>
<h2>查询运费</h2><h3 id="m0">com.edu.test.TransFeeTest#feeTest3</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [请求失败] but found [success]
	at com.edu.test.TransFeeTest.feeTest3(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m1">com.edu.test.TransFeeTest#feeTest5</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [请求失败] but found [success]
	at com.edu.test.TransFeeTest.feeTest5(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m2">com.edu.test.TransFeeTest#feeTest6</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [请求失败] but found [success]
	at com.edu.test.TransFeeTest.feeTest6(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m3">com.edu.test.TransFeeTest#feeTest7</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [请求失败] but found [success]
	at com.edu.test.TransFeeTest.feeTest7(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m4">com.edu.test.TransFeeTest#feeTest8</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [请求失败] but found [success]
	at com.edu.test.TransFeeTest.feeTest8(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m5">com.edu.test.TransFeeTest#feeTest9</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [请求失败] but found [success]
	at com.edu.test.TransFeeTest.feeTest9(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m6">com.edu.test.TransFeeTest#feeTest</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m7">com.edu.test.TransFeeTest#feeTest2</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m8">com.edu.test.TransFeeTest#feeTest4</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
</body>
</html>
